app/components/modal_options/modal_options.js

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+
+import {
+    View,
+    StyleSheet,
+    Dimensions,
+    Modal,
+    Text,
+    ScrollView,
+    TouchableOpacity,
+    TouchableWithoutFeedback
+} from 'react-native';
+
+import FormattedText from 'app/components/formatted_text';
+
+const {width} = Dimensions.get('window');
+const styles = StyleSheet.create({
+    overlayStyle: {
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        justifyContent: 'flex-end',
+        paddingBottom: 10,
+        paddingLeft: 5,
+        backgroundColor: 'rgba(0,0,0,0.3)'
+    },
+
+    optionContainer: {
+        borderRadius: 10,
+        width: (width - 10),
+        minHeight: 50,
+        backgroundColor: '#fff',
+        marginBottom: 5
+    },
+
+    cancelStyle: {
+        borderRadius: 10,
+        width: (width - 10),
+        height: 50,
+        justifyContent: 'center',
+        backgroundColor: '#fff',
+        padding: 8
+    },
+
+    cancelTextStyle: {
+        textAlign: 'center',
+        color: '#4E8ACC',
+        fontWeight: 'bold',
+        fontSize: 17
+    },
+
+    optionStyle: {
+        padding: 8,
+        height: 50,
+        justifyContent: 'center',
+        borderBottomWidth: 1,
+        borderBottomColor: '#999999'
+    },
+
+    optionTextStyle: {
+        textAlign: 'center',
+        fontSize: 17,
+        color: '#4E8ACC'
+    },
+
+    titleStyle: {
+        paddingTop: 8,
+        paddingHorizontal: 10,
+        height: 70,
+        borderBottomWidth: 1,
+        borderBottomColor: '#999999'
+    },
+
+    titleTextStyle: {
+        fontSize: 15,
+        color: '#7f8180'
+    }
+});
+
+export default class ModalOptions extends PureComponent {
```

Comment:
Sorry, I know there's a bunch of renaming that you'll have to do for this, but can you rename this to `OptionsModal`? Every time I see this, I keep thinking that it's something you pass into a modal to tell it how to look and behave. I think that's also consistent with modal names in the webapp and with things like containers that are `PostListContainer` instead of `ContainerPostList`